### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by JaqKent**
that Justify-content in "open" class do  lo que se le canta el pingo.

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# cuervodelnorte-frontend
+
+Cuervo del Norte Store Frontend web development
```
Comment (nathan-hyan) [inline README.old.md]: Is this file necessary?

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import NavBar from "./components/navbar";
```
Comment (nathan-hyan) [inline src/App.tsx]: Try absolute imports for this project 

https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8

File: src/components/navbar/NavItem/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./styles.module.scss";
+
+interface Props {
+  label?: string;
+  onClick: () => void;
+  isActive?: Boolean;
```
Comment (nathan-hyan) [inline src/components/navbar/NavItem/index.tsx]: Boolean with a capital B is a function, make it "boolean" to avoid future issues.

File: src/components/navbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+import NavItem from "./NavItem";
+import menu from "assets/Hamburger Icon.svg";
+import cart from "assets/Cart Icon.svg";
```
Comment (nathan-hyan) [inline src/components/navbar/index.tsx]: Avoid spaces in file names

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Great job so far, a few comments and a reminder: Remove unused files, such as tests, testConfigs and all React boiler plate (such as 'public/logo...' images)

File: README.old.md (hunk 1 above)
Comment (JaqKent) [reply README.old.md]: done

File: src/App.tsx (hunk 2 above)
Comment (JaqKent) [reply src/App.tsx]: done

File: src/components/navbar/NavItem/index.tsx (hunk 3 above)
Comment (JaqKent) [reply src/components/navbar/NavItem/index.tsx]: done

File: src/components/navbar/index.tsx (hunk 4 above)
Comment (JaqKent) [reply src/components/navbar/index.tsx]: done

**Review by nathan-hyan — APPROVED** [review]
🥇
